commands/generate.js

```diff
@@ -3,7 +3,17 @@ import fs from 'fs';
 import path from 'path';
 
 const init = () => {
-	const configFile = path.join(atom.project.getPaths()[0], '.editorconfig');
+	let basePath = '';
+	if (atom.project.getPaths().length > 0) {
```

Comment:
`atom.project.getPaths()`, you can call that once, store the results, then do the check on the `.length` here and the `[0]` element on the next line without re-calling the function.